src/structures/point.py

```diff
@@ -67,17 +67,17 @@ def from_tensor(inp: torch.Tensor) -> 'Point':
         assert len(list(inp.size())) == 1
         return Point(*[int(torch.round(inp[i])) for i in range(list(inp.size())[0])])
 
-    def __add__(self, other: 'Point') -> 'Point':
-        return Point(*np.add(self.__values, other.__values))
+    def __add__(self, other: Any) -> 'Point':
+        return Point(*np.add(self.__values, other.__values if type(other) is Point else other))
```

Comment:
This is nice because it allows us to add things like lists or tuples to Points, but shouldn't we also check to make sure we are adding values of a type tuple (or an np array). I think np.add for example adds an array and a single value without raising an error, which I don't know if it is the behavior you're looking for.

Reply:
That's precisely the behaviour I was looking for, reason why I changed it in the first place. I don't think there is a need to do any further checks. Otherwise, the function will start having too much overhead.